ldap/servers/slapd/daemon.c

```diff
@@ -206,7 +206,9 @@ static int time_shutdown = 0;
 char *
 disk_mon_get_mount_point(char *dir)
 {
+    struct mntent mntbuf;
     struct mntent *mnt;
+    char buf[4096 + 1]; /* enough for 2 paths */
```

Comment:
Just being paranoid, but should this zero'd out?

    char buf[4096 + 1] = {0};